### PR TITLE
added falcon sportsbook and additional helper functions

### PR DESCRIPTION
**Pull request by d-black1415**

### CODE REVIEW

File: src/sportsbooks/falcon_nfl_scrape.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        super().__init__()
+        self.book_name = 'Falcon'
+        
+    def login_and_retrieve_nfl_page(self):
+        with requests.Session() as s:
+            falcon_creds = CredentialReader.read_cred_row(FALCON_CRED_ROW_IDX)
+            FALCON_LOGIN_FORM['account'] = falcon_creds[0]
+            FALCON_LOGIN_FORM['password'] = falcon_creds[1]
+            
+            login_req = s.post(FALCON_LOGIN_URL, data = FALCON_LOGIN_FORM)
```
Comment (davidiola) [inline src/sportsbooks/falcon_nfl_scrape.py:20]: is this variable used anywhere? if not,
`s.post(FALCON_LOGIN_URL, data = FALCON_LOGIN_FORM)`

File: src/util/helpers.py [hunk 2]
```diff
@@ -41,6 +43,30 @@ def total_line(text):
...
+    if book_name == 'Falcon':
+        home_series = dict()
+        home_series['Month'], home_series['Day'] = convert_integer_date(game['gmdt'])
+        home_series['Team_ID'] = game['hnum']
+        home_series['Internal_ID'] = game['idgm']
+        home_series['Team'] = game['htm']
+        home_series['Spread'] = game['GameLines'][0]['hsprdt']
+        home_series['Spread_Line'] = game['GameLines'][0]['hsprdoddst']
+        home_series['Total'] = game['GameLines'][0]['ovt']
+        home_series['Total_Line'] = game['GameLines'][0]['ovoddst']
```
Comment (davidiola) [inline src/util/helpers.py:56]: what do these hardcoded strings refer to/should we define them as constants

File: src/util/helpers.py [hunk 3]
```diff
@@ -93,3 +119,26 @@ def fill_dates(df, first_row = False):
...
+    '07': 'Jul',
+    '08': 'Aug',
+    '09': 'Sep',
+    '10': 'Oct',
+    '11': 'Nov',
+    '12': 'Dec'
+    }
+
+# Converts date from 'yyyymmdd' to month and day
+def convert_integer_date(date, month_dict = month_dict):
```
Comment (davidiola) [inline src/util/helpers.py:139]: i don't think we need to pass `month_dict` in as a param here

File: src/util/helpers.py [hunk 4]
```diff
@@ -93,3 +119,26 @@ def fill_dates(df, first_row = False):
...
+    new_date = datetime.strptime(date.strip(), '%Y%m%d').strftime('%m/%d/%Y')
+    first_slash = new_date.find('/')
+    second_slash = new_date.find('/', first_slash + 1)
+    return month_dict[new_date[:first_slash]], new_date[first_slash + 1: second_slash]
```
Comment (davidiola) [inline src/util/helpers.py:143]: alternatively, could use split to split `new_date` into an array of parts using "/" as the delimeter


```suggestion
new_date_parts = new_date.split('/')
return month_dict[new_date_parts[0]], new_date_parts[1]
```

File: src/sportsbooks/falcon_nfl_scrape.py (hunk 1 above)
Comment (d-black1415) [reply src/sportsbooks/falcon_nfl_scrape.py:20]: ya I didn't even realize that I didn't need the cookies

File: src/util/helpers.py (hunk 2 above)
Comment (d-black1415) [reply src/util/helpers.py:56]: they are just the locations of the information for each game that is JSON formatted. we can make constants but I think they are pretty self explanatory. Up to you

File: src/util/helpers.py (hunk 3 above)
Comment (d-black1415) [reply src/util/helpers.py:139]: you are correct

**Pull request merged**